src/cds-hooks/service.ts

```diff
@@ -132,47 +142,58 @@ export interface HookResponse {
  * https://cds-hooks.org/specification/current/#prefetch-template
  */
 interface PrefetchTemplate {
-  [key: string]: string
+  [key: string]: string;
+}
+
+/**
+ * An extension is an additional information exchange specification based on
+ * common requirements across healthcare. FHIR expects that additional requirements
+ * are implemented as extensions.
+ * See: https://cds-hooks.org/specification/current/#extensions
+ * See: https://www.hl7.org/fhir/extensibility.html
+ */
+interface Extension {
+  [key: string]: string;
```

Comment:
the value here should probably be `any` instead of string right, because each key could itself be an object

Reply:
Ooo yeah 100%, in the example it can be an array as well.